doc/sphinx/source/quickstart/installation.rst

```diff
@@ -554,15 +406,16 @@ To do this, run the tool with:
 If everything was installed properly, ESMValTool should have printed a
 help message to the console.
 
-**MacOSX note:** some recipes may depend on the OpenMP library, which does not
-install via mamba on MacOSX. Instead run
+.. note::
+    **MacOS users:** some recipes may depend on the OpenMP library, which does not
+    install via mamba on MacOS. Instead run
 
-.. code-block:: bash
+    .. code-block:: bash
 
-    brew install libomp
+        brew install libomp
 
-to install the library with Homebrew. In case you do not have Homebrew, follow
-installation instructions `here <https://brew.sh/>`__.
+    to install the library with Homebrew. In case you do not have Homebrew, follow
+    installation instructions `here <https://brew.sh/>`__.
```

Comment:
can't they use Macports for that? If so, let's not impose install methods on Mac-ers, they're special :grin:

Reply:
I have no clue, haven't had a Mac for almost a decade now.. but if you think this can be improved you're welcome to do it!

Reply:
maybe add it like "your preferred installation method for OSX packages?"